### ISSUE TITLE
Error: 0x006E0968

### ISSUE THREAD

**Issue by Gthbacc** [juicermv/GTA-GPS-Redux#37]
Hey. Big fan of your fork. There seems to be only one issue and I encounter it when I enter police bikes:

According to crashinfo, The game crashes when you use this mod alongside the framerate vigilante mod. In the regular gps mod (Juniors) this issue is fixed and crashinfo suggests the player use the updated version of that mod to fix it. I tested this and the issue did not occur with the original mod.

Please keep an eye on this issue. Thanks in advance.

**Comment by juicermv** [comment]
Hey, do you mind uploading a screenshot of the error itself and perhaps a dump file if there is one? Thanks. I'll try reaching out to Junior.

**Comment by juicermv** [comment]
Also, I myself use framerate vigilante with the mod and have not encountered this issue so far.

Make sure you have the latest version of framerate vigilante installed. The error explanations given with CrashInfo are usually just guesses and aren't always correct. 

I'd make sure no other mod is possibly interfering.

**Comment by Gthbacc** [comment]
Hey, thanks for the quick response. I'm also aware that the solutions given in crashinfo are mere suggestions based on guesses but weirdly enough, this seems rather accurate. I started building a modded gta about a month ago and i kept testing and progressing through the story as I added most. Whats weird is that this issue seems rather recent! I've already beaten the game and I dont remember having this issue throughout my gameplay at all. And GPS Redux & Framerate Vigilante were among the first mods I used.
Very strange indeed. I tried removing some mods that I added recently but the isdue still persisted. I'll add the screenshot and the log in a moment.

**Comment by Gthbacc** [comment]
`GTA SA 1.0.0.0 US
Unhandled exception at 0x006E0968 in gta_sa.exe (+0x2e0968): 0xC0000005: Access violation reading location 0x1A9DC221.
    Register dump:
        EAX: 0x00000001  EBX: 0x00000002  ECX: 0x19E68CB8  EDX: 0x00000000  
        EDI: 0x00000001  ESI: 0x19E68CB8  EBP: 0x19E68D10  EIP: 0x006E0968  
        ESP: 0x0177EE00  EFL: 0x00210246  CS: 0x00000023   SS: 0x0000002B   
        GS: 0x0000002B   FS: 0x00000053   ES: 0x0000002B   DS: 0x0000002B   
        
    Stack dump:
        0x0177EE00:  00000001 19E68CB8 006BBC05 00000001 4723C594 19E68CB8
        0x0177EE18:  0177EFDC 00000800 00000000 3C078B73 3ECDF206 3F6A6089
        0x0177EE30:  00000000 00FDBF49 BF71BFFC 3EA8708D 00000000 3F6B851F
        0x0177EE48:  3AC49BA5 BEA8708C BF71BFFA 00000000 3F618F07 00000000
        0x0177EE60:  378B157F 38111270 BA904D78 00630E68 3F6E147B 3F800000
        0x0177EE78:  392E99D6 38421F7E BA86210D 3AC49BA5 00000000 00000002
        0x0177EE90:  B7C1FD21 3F800000 0077EEF8 3F8A3D70 BE9ACE20 BF5F1DD1
        0x0177EEA8:  BF102F81 3EAD43A2 3F77B3B2 BF0F0026 BF71BA3F 3EA86C8E
        0x0177EEC0:  00000000 5C357046 0003931A FD1C35C8 80000000 80000000
        0x0177EED8:  BF800000 80000000 80000000 BF800000 80000000 80000000
        base: 0x01580000   top: 0x0177EE00   bottom: 0x01780000
        
    Backtrace (may be wrong):
        =>0x006E0968 _ZN8CVehicle19ProcessSirenAndHornEb+0x18 in gta_sa.exe (+0x2e0968) (0x0177EE04) 
          0x006BBC05 _ZN5CBike14ProcessControlEv+0x29b5 in gta_sa.exe (+0x2bbc05) (0x0177EE68) 
          0x00630E68 _ZN15CTaskSimpleDead10ProcessPedEP4CPed+0x868 in gta_sa.exe (+0x230e68) (0x0177EE6C) 
          0x3F6E147B in unknown (+0x3f6e147b) (0x0177EF6C) 
          0x004E6B03 sub_4E6AE0+0x23 in gta_sa.exe (+0xe6b03) (0x0177EF74) 
          0x005E9529 _ZN4CPed14ProcessControlEv+0x859 in gta_sa.exe (+0x1e9529) (0x0177EF7C) 
          0x0073DB6A _ZN7CWeapon6UpdateEP4CPed+0x2a in gta_sa.exe (+0x33db6a) (0x0177EFA4) 
          0x00568642 _ZN6CWorld7ProcessEv+0x1a2 in gta_sa.exe (+0x168642) (0x0177EFB4) 
          0x0053C09A _ZN5CGame7ProcessEv+0x1ba in gta_sa.exe (+0x13c09a) 
        
    
`

![https://i.ibb.co/B2vd2gQ/Screenshot-9098.png](url)

**Comment by juicermv** [comment]
Which version of the mod are you using? Does the issue get resolved once you remove/disable GPS Redux?

**Comment by Gthbacc** [comment]
Both mods are the latest releases. Yes, once GPS Redux is removed, the issue is gone. Also, the crash only happens when using a police bike. No other vehicle with sirens has this issue afaik.

**Comment by juicermv** [comment]
This is very strange. Could you send me a full list of every mod you have installed? This is most likely an incompatibility with something else

**Comment by Gthbacc** [comment]
Sadly, I'm currently away from my PC. I could send the full list but it's +130 mods and I dont intend to Waste your time. It certainly is strange cause I know for a fact that I've used police bikes during my playthrough and it never crashed. It started happening only after I beat every mission. If you could tell me what function in the game is it exactly that you might think is causing the crash, or what type of mod could be doing this, I can come up with a fix.

**Comment by juicermv** [comment]
From looking at the stack trace you provided it just doesn't look like this is caused by any code related to GPS Redux, that's the thing. I've tried reaching out to Junior as well via Discord but he does not seem to accept DMs so I gotta pray he answers my friend request.

Either way, again by referencing the stack trace I'd guess it is something to do with ImVeht or any of those other vehicle-related mods.

**Comment by Gthbacc** [comment]
Found the issue! The TruckTrailer.asi mod was apparently interfering with the mod. I did some digging, too. Found a thread on gtaforums where the op solved the issue by removing Framerate Vigilante. Found another thread on the mixmods forums where it was stated that apparently TruckTrailer.asi was interfering with the First Person Mod. This lead me to figure the issue out.
Thank you so much for your patience and quick response. Best of luck to you in your future endeavors.
